packages/ember/index.js

```diff
@@ -203,8 +203,6 @@ Object.defineProperty(Ember, 'NAME_KEY', {
   },
 });
 
-Ember._Cache = utils.Cache;
```

Comment:
can you share where this is duplicated?

Reply:
sorry, on the line 283
https://github.com/emberjs/ember.js/blob/master/packages/ember/index.js#L283